flows/actions/classify_nlu.go

```diff
@@ -0,0 +1,130 @@
+package actions
+
+import (
+	"encoding/json"
+
+	"github.com/nyaruka/goflow/assets"
+	"github.com/nyaruka/goflow/flows"
+	"github.com/nyaruka/goflow/flows/events"
+	"github.com/pkg/errors"
+)
+
+func init() {
+	registerType(TypeClassifyNLU, func() flows.Action { return &ClassifyNLUAction{} })
+}
+
+const (
+	classificationCategorySuccess = "success"
+	classificationCategorySkipped = "skipped"
+	classificationCategoryFailure = "failure"
+)
+
+var classificationCategories = []string{classificationCategorySuccess, classificationCategorySkipped, classificationCategoryFailure}
+
+// TypeClassifyNLU is the type for the classify NLU action
+const TypeClassifyNLU string = "classify_nlu"
```

Comment:
bike sheddy but do we need the `_nlu`? Why not just `classify`?

Reply:
Was trying to follow the pattern of action names being verb+noun.. `send_msg` vs `reply`, `enter_flow` vs `flow` etc.. `classify` feels a little vague...

Reply:
`nlu` isn't much of a noun though. `classify_text`?

Reply:
That works for me